README.md

```diff
@@ -4,17 +4,24 @@ riscv-tests
 About
 -----------
 
-This repository hosts unit tests for RISC-V processors.
+This repository hosts unit tests for RISC-V processors including the instructions of the Xpulpimg extension subsets as defined [here](https://iis-people.ee.ethz.ch/~balasr/pulp-ext/pulp.html).
+
+This repository can be used to do behavioral [Spike](https://github.com/pulp-platform/riscv-isa-sim) simulations of instructions. This allows verification of new instructions or to find out the validity of a test case.
+
+The tests can also be used for RTL simulation (outside of the scope of
+`riscv-tests`).
```

Comment:
So we can finally get rid of the `snitch_isa.mk` in `riscv-tests`, and deal with RTL-related stuff outside, right?